src/main/java/mate/dao/jdbc/ManufacturerJdbcImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.dao.ManufacturerDao;
+import mate.exception.DataProcessingException;
+import mate.lib.Dao;
+import mate.model.Manufacturer;
+import mate.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerJdbcImpl implements ManufacturerDao {
+
+    private static Optional<Manufacturer> createNewManufacturer(ResultSet resultSet) {
```

Comment:
- why is this method static?
- private methods should be at the end of the file after all public, or after the public method where it is used.
- Optional is redundant for this method